src/View/Helper/FormHelper.php

```diff
@@ -169,7 +167,17 @@ public function input($fieldName, array $options = [])
                 if ($options['inline']) {
                     $options['label'] = $this->injectClasses($options['type'] . '-inline', (array)$options['label']);
                 }
-
+                break;
+            case 'radio':
+                $isInline = (isset($options['inline']) && $options['inline'] === true);
+                if (!$isInline || $this->_align === 'horizontal') {
+                    $this->templater()->remove('radioFormGroup');
+                }
+                if (!$isInline) {
+                    $this->templater()->add([
+                        'nestingLabel' => '<div class="radio">' . $this->templater()->get('nestingLabel') . '</div>'
```

Comment:
I would keep this otherwise it's more complicate to handle the `nestingLabel`. Wrap the `nestingLabel` in `<div class="radio"></div>` is a special case for radios here. If you place the the `netstingLabel` to the templates directly you need to change the `nestingLabel` for all other inputs conditionally in the helper. Besides the wrapper is forced by bootstrap so I think there is no reason for the user to change the template.

The `nestingLabel` is also customizable this way.